kaniko.go

```diff
@@ -82,13 +95,32 @@ func (p Plugin) Exec() error {
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--cache-ttl=%d", p.Build.CacheTTL))
 	}
 
+	if p.Build.DigestFile != "" {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--digest-file=%s", p.Build.DigestFile))
+	}
+
 	cmd := exec.Command("/kaniko/executor", cmdArgs...)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	trace(cmd)
 
 	err := cmd.Run()
-	return err
+	if err != nil {
+		return err
+	}
+
+	if p.Build.DigestFile != "" {
+		content, err := ioutil.ReadFile(p.Build.DigestFile)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%s\n", err)
+		}
+		err = artifact.WritePluginArtifactFile(artifact.Docker, p.Artifact.ArtifactFile, p.Artifact.Registry, p.Artifact.Repo, string(content), p.Artifact.Tags)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%s\n", err)
```

Comment:
Ditto

Reply:
Im not returning those errors, if i return error plugin exec will be marked as failed.